apps/meteor/tests/unit/app/apps/server/mocks/data/messages.data.js

```diff
@@ -105,3 +109,44 @@ export const appMessageInvalidRoomMock = {
 		t: 'uj',
 	},
 };
+
+const testUsername = faker.internet.userName();
+const testUserId = faker.database.mongodbObjectId();
+export const exportMessagesMock = [
+	createFakeMessage({ t: 'uj', u: { _id: testUserId, username: testUsername }, msg: testUsername }),
+	createFakeMessage({
+		msg: '',
+		file: {
+			_id: 'txt-file-id',
+			name: 'test.txt',
+			type: 'text/plain',
+			size: 29,
+			format: '',
+		},
+		attachments: [
+			{
+				type: 'file',
+				title: 'test.txt',
+				title_link: '/file-upload/txt-file-id/test.txt',
+			},
+		],
+	}),
+	createFakeMessage({
+		msg: '',
+		file: {
+			_id: 'txt-file-id',
+			name: 'test.txt',
+			type: 'text/plain',
+			size: 29,
+			format: '',
+		},
+		attachments: [
+			{
+				type: 'file',
```

Comment:
also this one doesn't have a title, is that good?

Reply:
Yep, I'm testing specifically what happens when no `title` is provided